scaling/src/test/java/io/quarkus/ts/openshift/scaling/ScalingOpenShiftIT.java

```diff
@@ -38,91 +43,81 @@ public void scaleBack() {
+        int REPLICAS = EXPECTED_SCALE_UP;
 
-        openShiftUtil.scale(DC_NAME, 2);
-
-        assertThat(openShiftUtil.countReadyReplicas(DC_NAME)).isEqualTo(2);
-
-        Set<String> uniqueIds = Collections.newSetFromMap(new ConcurrentHashMap<>());
-        await().atMost(1, TimeUnit.MINUTES).untilAsserted(() -> {
-            String uniqueId =
-                    given()
-                            .when().get("/scaling")
-                            .then()
-                            .statusCode(OK.getStatusCode())
-                            .extract().asString();
-            uniqueIds.add(uniqueId);
-
-            assertThat(uniqueIds).hasSize(2);
-        });
+        givenResourcePath("/scaling");
+        whenScaleTo(EXPECTED_SCALE_UP);
+        thenCheckReplicasAmount(EXPECTED_SCALE_UP);
+        whenMakeRequestTo("/scaling", REPLICAS);
     }
 
+
     /**
      * Workflow:
      * * Scale to two replicas and verify that both of the replicas are responding.
      * * Scale down to a single replica.
      * * Execute an arbitrary minimal sample of requests and verify that all are served by the same replica.
      */
     @Test
-    public void scaleDownTest() {
-        openShiftUtil.scale(DC_NAME, 2);
-
-        assertThat(openShiftUtil.countReadyReplicas(DC_NAME)).isEqualTo(2);
-
-        Set<String> uniqueIds = Collections.newSetFromMap(new ConcurrentHashMap<>());
-        await().atMost(1, TimeUnit.MINUTES).untilAsserted(() -> {
-            String uniqueId =
-                    given()
-                            .when().get("/scaling")
-                            .then()
-                            .statusCode(OK.getStatusCode())
-                            .extract().asString();
-            uniqueIds.add(uniqueId);
-
-            assertThat(uniqueIds).hasSize(2);
-        });
-
-        openShiftUtil.scale(DC_NAME, 1);
-
-        assertThat(openShiftUtil.countReadyReplicas(DC_NAME)).isEqualTo(1);
-
-        uniqueIds.clear();
-        for (int i = 0; i < 100; i++) {
-            String uniqueId =
-                    given()
-                            .when().get("/scaling")
-                            .then()
-                            .statusCode(OK.getStatusCode())
-                            .extract().asString();
-            uniqueIds.add(uniqueId);
-        }
-
-        assertThat(uniqueIds).hasSize(1);
-
-        assertThat(openShiftUtil.countReadyReplicas(DC_NAME)).isEqualTo(1);
+    public void scaleDownTest() throws InterruptedException {
+
+        int REPLICAS = EXPECTED_SCALE_UP;
+        givenResourcePath("/scaling");
+        whenScaleTo(EXPECTED_SCALE_UP);
+        thenCheckReplicasAmount(EXPECTED_SCALE_UP);
+        whenMakeRequestTo("/scaling", REPLICAS);
+
+        REPLICAS = EXPECTED_SCALE_DOWN;
+        whenScaleTo(EXPECTED_SCALE_DOWN);
+        thenCheckReplicasAmount(EXPECTED_SCALE_DOWN);
+        whenMakeRequestTo("/scaling", REPLICAS);
```

Comment:
I personally read much clearer the numbers directly instead of EXPECTED_SCALE_DOWN or EXPECTED_SCALE_UP.